types/jkiten-react-date-range/index.d.ts

```diff
@@ -0,0 +1,344 @@
+// Type definitions for jkiten-react-date-range 1.4
+// Project: https://github.com/OnionFarm-Corp/jkiten-react-date-range
+// Definitions by: Jkiten <https://github.com/OnionFarm-Corp>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
```

Comment:
```suggestion
```
that has no longer a meaning here, TS4.3 is the lowest one, so just remove probably